lib/src/test/java/io/ably/lib/test/rest/HttpTest.java

```diff
@@ -147,6 +163,118 @@ public Http setUrlArgumentStack(List<String> urlArgumentStack) {
 	}
 
 	/**
+	 * Validates that fallbacks are disabled when ClientOpitons#fallbackHosts are set to null and the only host used is Defaults#HOST_REST
```

Comment:
This description is wrong. According to http://docs.ably.io/client-lib-development-guide/features/#TO3k6 fallback hosts are only disabled if an empty array is provided. Setting fallbackHosts to null, is effectively the same as not providing them, which AFAICT would break every client in that they will no longer us fallback hosts by default (unless ClientOptions sets that default of course, which I assume it does).  If `ClientOptions#fallbackHosts` default is populated, then perhaps this is OK to be honest. @paddybyers WDYT?